doc/use_cases/tpm/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Trusted Platform Module - TPM"
+---
+
+## Overview
+
+OpenTitan can be used to implement the full Trusted Platform Module (TPM) 2.0
+specification to meet client and server platform use cases. When used as a TPM,
+OpenTitan is provisioned with an endorsement seed and RSA and ECDSA endorsement
+certificates (EK). TPM commands are served over either SPI or I2C device
+peripherals.
+
+## Certification Requirements
+
+*   [ANSSI-CC-PP-2018/03](https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClient_PP_1p3_for_Library_1p59_pub_29sept2021.pdf)
+    Protection Profile Client Specific TPM[^1]. The minimum assurance level for
+    this Protection Profile (PP) is EAL 4 augmented with ALC\_FLR.1 and
+    AVA\_VAN.4.
+
+    *   ALC\_FLR.1: Basic flaw remediation. The developer provides flaw
+        remediation procedures to the Target of Evaluation (TOE) developers.
+    *   AVA\_VAN.4: Methodical vulnerability analysis. Methodical vulnerability
+        analysis is performed by the evaluator to identify the presence of
+        potential vulnerabilities. Penetration testing is performed by the
+        evaluator with a _moderate _attack potential.
+
+## Minimum Crypto Algorithm Requirements
+
+*   TRNG: At least one internal entropy source is required. The entropy source
+    and collector should provide entropy to the state register in a manner that
+    is not visible to an outside process. The entropy collector should regularly
+    update the state register with additional, unbiased entropy.
+*   Hash Algorithms:
+    *   An approved hash algorithm with approximately the same security strength
+        as its strongest asymmetric algorithm. For OpenTitan the target is
+        SHA2-256 (hardware), SHA3-384 (software implementation).
```

Comment:
Sorry for the late comment, but wanted to clarify that cryptolib will probably use a **software** SHA2-256 for most cases. TPM sets up hashes with an init/update/final interface that requires the ability to save/restore hash state so multiple processes can hash in an interleaved manner, and the current hardware doesn't have this capability. Also, did you mean SHA3-384 here or SHA2-384? We will have SHA2-384 and SHA2-512 (software) as well as hardware SHA3-{224,256,384,512} from KMAC (but again, not compatible with the init/update/final TPM interface unless only one process is using the SHA3 at a time).

Reply:
Hi @jadephilipoom, thanks for taking a look at this PR! I created #14537 to fix these issues. Let's continue the discussion there. Thanks